platforms/mixin-common/README.md

```diff
@@ -1,4 +1,4 @@
 # mixin-common
 
-This project contains mixins shared between Forge, Fabric and Quilt, as
+This project contains mixins shared between Forge, Fabric and formerly Quilt, as
```

Comment:
Change this line to "This project contains mixins shared between Forge and Fabric, as"?